Assets/MixedRealityToolkit.Examples/Experimental/Demos/UX/BoundsControl/Scripts/BoundsControlRuntimeExample.cs

```diff
@@ -12,9 +13,9 @@
 
 namespace Microsoft.MixedReality.Toolkit.Examples.Experimental.Demos
 {
+    [AddComponentMenu("Scripts/MRTK/Experimental/Examples/BoundsControlRuntimeExample")]
```

Comment:
Do we expect / wish to encourage adding this script via the component menu? Is there precedent for MRTK doing so?

Reply:
Tbh I wasn't sure about that, I kept on removing / readding that line while I was patching up things :D - I think we shouldn't encourage ppl to create this via the component menu but the old demo for bounding box had the entry, so that's why i in the end went for leaving it in. Now that you think this is odd as well I will remove it. Thanks!